docs/Installation.md

```diff
@@ -52,6 +52,12 @@ for Zonemaster::Backend, see the [declaration of prerequisites].
 > **Note:** In addition to the normal dependencies, the post-installation
 > smoke test instruction assumes that you have curl installed.
 
+Optionally install jq (only needed for the post-installation smoke test):
+
+```sh
+sudo yum install jq
+```
+
 ## 3. Installation on CentOS
 
 ### 3.1 Install Zonemaster::Backend and related dependencies (CentOS)

```

Comment:
To be consistent with Debian and FreeBSD, under this headline, just after the "note" about Zonemaster::LDNS and Zonemaster::Engine add:

Optionally install Curl and jq (only needed for the post-installation smoke test):

```sh
sudo yum install curl jq
```

Reply:
The scope of this PR is to update the smoke test. It's not about fixing all problems we can find in the installation instruction.

Reply:
I moved the misplaced jq installation step here. I'm not adding curl here because it isn't a new dependency. I haven't checked but I'm assuming it's installed by default on CentOS.